api/iTechArt.Hotels.Api.csproj

```diff
@@ -18,4 +18,8 @@
     <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.13.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Resources\Images\" />
+  </ItemGroup>
+
```

Comment:
Do you need this?

Reply:
No, resources folder is not a project part